tests/jballerina-integration-test/src/test/resources/packaging/spi/TestProject/src/module/funcs.bal

```diff
@@ -1,5 +1,11 @@
 import ballerina/io;
+import ballerinax/java;
 
 public function main() {
-    io:println("Hello World");
+    io:println(getDriversAsString());
 }
+
+public function  getDriversAsString() returns string = @java:Method {
+      class:"PrintDrivers"
```

Comment:
Where is the PrintDrivers class loaded?

Reply:
its test jar i created jballerina-integration-test/src/test/resources/packaging/spi/TestProject/libs/drivers.jar